detection_rules/packaging.py

```diff
@@ -22,6 +24,8 @@
 PACKAGE_FILE = get_etc_path('packages.yml')
 NOTICE_FILE = get_path('NOTICE.txt')
 
+CURRENT_PACKAGE_VERSION = load_dump(PACKAGE_FILE)['package']['name']
```

Comment:
one alternative so this doesn't barf if the yaml is invalid:
make this a `@cached` function instead